client/app/containers/ConnectorManagement.jsx

```diff
@@ -187,6 +194,20 @@ export default class ConnectorManagement extends Component {
     return defaults
   }
 
+  checkProperties (connector) {
+    // remove invalid dataType declaration
+    // (valid = present in AUTHORIZED_DATATYPE)
+    let checkedTypes = []
+    for (let dType of connector.dataType) {
+      if (AUTHORIZED_DATATYPE.find(function (t) {
```

Comment:
Great! I was wondering how to improve this piece of code, here it is 👍.

I also agree about the `sanitize` function, I just not wanted to name a function specific to data type here, we could use it to check other properties later, so it's the right name for here.

 Thanks @gregorylegarec for your feedbacks 🎉. I will do these changes.